CONTRIBUTING.md

```diff
@@ -145,7 +145,7 @@ When submitting a pull request:
 ### Acceptance Criteria
 
 For a pull request to be merged, it must:
-- Receive approval from at least 2 reviewers
+- Receive approval from at least 2 reviewers ([team members](TEAM.md))
```

Comment:
```suggestion
- Receive approval from at least 2 reviewers with write access
- Receive no objections from reviewers with write access
```

Reply:
That is not the same. It's possible for non-team-members to have write access.